### ISSUE TITLE
Fira Code style Arrow combinations

### ISSUE THREAD

**Issue by vikky49** [JetBrains/JetBrainsMono#256]
This is just a general question , recently Fira code implemented different style arrow combinations. Any chance of that being implemented here . I think that also helps remove various ligatures 

Image here 
https://user-images.githubusercontent.com/285292/82168184-74dbee00-98be-11ea-9743-1a707e64018e.png
<img width="961" alt="82168184-74dbee00-98be-11ea-9743-1a707e64018e" src="https://user-images.githubusercontent.com/180512/84810411-2b1b1c00-afc0-11ea-9e4f-dce2eb214978.png">

**Comment by philippnurullin** [comment]
Yes, we are following the good work Nikita is doing with Fira Code.
This looks impressive, but i don't see a big improvement in everyday work with this feature. We still have a lot to do before adding it.
Thanks for suggestion.

**Comment by MartenBE** [comment]
Is there any update on this? Fonts like Fira Code and Cascadia Code from Microsoft have this functionality. It is very handy in terminal output scenario's.
